arthimetic-operators.js

```diff
@@ -28,3 +34,19 @@ Task 1:
    5. Divide two numbers and round the result to the nearest integer before logging it to the console.
 *******************************************************************************/
 // TODO: ADD YOUR CODE BELOW
+
+let x= 4-3
+console.log(x)
+
+let y= 1*3*5*7
+console.log(y);
+
+let rectangular= 3*5*7
+console.log(rectangular); 
+let declare=9.99%0.20
+
```

Comment:
Declare another variable discount and assign it the value **0.20** and calculate the discounted price and log the result to the console.